packages/edit-site/src/components/global-styles/screen-block-list.js

```diff
@@ -45,6 +45,11 @@ function BlockMenuItem( { block } ) {
 }
 
 function ScreenBlockList() {
+	const inserterItems = useSelect( ( select ) => {
+		const { getInserterItems } = select( blockEditorStore );
+		return getInserterItems();
```

Comment:
We shouldn't use `getInserterItems` selector as it also fetches block variations and reusable blocks and in Global Styles we don't want this. We should use `getBlockTypes` and sort them like we do in the [getInserterItems](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/store/selectors.js#L1694).

For reference the original implementation was using `Array.prototype.sort` but it was [changed here](https://github.com/WordPress/gutenberg/pull/34078/files) because of implementation details in the javascript engine(mobile), so I think we can use `reduce` as well.

Reply:
Thanks so much for the good catch! I fixed the PR as per your comment. 😃